index.tsx

```diff
@@ -238,85 +236,111 @@ function Folder({
   }, [content])
 
   return (
-    <NavigationStack>
```

Comment:
I thought that I have to include a NavigationStack at the root of a component when the component is used in Navigation.present or as target of a NavigationLink. Is this not the case?